impeller/aiks/aiks_unittests.cc

```diff
@@ -449,6 +449,66 @@ TEST_P(AiksTest, CanRenderLinearGradientWithDitheringEnabled) {
   CanRenderLinearGradientWithDithering(this, true);
 }  // namespace
 
+namespace {
```

Comment:
Just `static` instead of the anonymous namespace?

Reply:
Done. Also applied to my other gradient test cases since they are quite similar.